### ISSUE TITLE
Midterm Report Peer Review

### ISSUE THREAD

**Issue by noriolg** [Meiqi-Wu/vaccine-discovery#7]
The project is aiming to predict the presence of a certain antibody for combinations of different features (namely peptides combined with parent proteins). The overall process followed and the steps taken to treat the data are clearly explained and shown in the report. It is a great idea with clear amounts of work behind and the report demonstrated that.

Three positive aspects of the project are:
* Steps taken in data preprocessing are explained in a concise and clear way. It helps with following the technical procedure and thought process that went on behind the scenes.

* Very clear and insightful full EDA. Plots help understand the relevance of each feature regarding the target label and if I were to be more knowledgeable on the topic, I am sure it would provide great amounts of information.

* It is clear that a lot of time has been invested into the project as demonstrated by the fact that they have tried various models and listed why they think are not suitable for the current problem and dataset.

Three possible areas of improvement are:
* I would make sure to include an introductory paragraph with an explanation of variables, what they mean, and how they relate to each other. I found that in the project proposal but I think it would also be helpful for people unfamiliar with the subject (such as myself) to have a 1 paragraph introduction to the project terminology and goals.

* I missed conclusions drawn from the provided plots. What do they tell us and what do the different data distributions imply? Some of these questions were addressed for the first group of plots but not for the second one (or at least not fully).  It might be that I am misunderstanding it but I think some more explanation regarding the plots' insights would be helpful.

* One final thing that could be improved is some perspective on the findings. I think that the model building and information obtained are there and it was done exhaustively and with good results. However, if in Future Steps, there were some explanations on how this might be used or the possible applications of this model I think it could be helpful.

In conclusion, I think this project was tackled in a very profound manner, and the method that was followed helps reach the desired conclusions. Although it might be hard to understand for people not familiar with the field, I struggled to find any technical or model-related problems with the approach taken by this project group.

Good luck!

Nico
